File: C/1.c
#include <stdio.h>

// Returns the sum of multiple k of n, where n * k < 1000
int sum(int n) {
    int count = 1000 / n;
    return count * (n + count * n) / 2;
}

int main(int argc, char *argv) {

    printf("%d", sum(3) + sum(5) - sum(15) - 1000);
    
    return 0;
}

File: JavaScript/1.js
(function () {
    // Returns the sum of multiples (k) of n, where n * k < 1000
    var s = function (n) {
            var i, sum;
            for (i = n, sum = 0; i < 1000; sum += i, i += n);
            return sum;
        },
        m3 = s(3),
        m5 = s(5),
        m15 = s(15);
    console.log(m3 + m5 - m15);
})();